semperfiwebdesign/all-in-one-seo-pack
# Refactor help text strings for XML Sitemap menu

Issue #2499

## Proposed changes

Refactors help text strings for XML Sitemap menu to optimize strings for translation.

## Types of changes

- Improves existing code

## Checklist


## Testing instructions

Check the help text for the "Frequency" and "Priority" settings in the XML Sitemap menu and make sure it looks good.

## Further comments

PR for "Frequency" & "Priority" settings in Video Sitemap menu can be found here - https://github.com/semperfiwebdesign/aioseop-pro/pull/496
